code/modules/events/crystal_event.dm

```diff
@@ -48,9 +47,100 @@ This section is for the event controller
 	var/obj/machinery/destabilized_crystal/dest_crystal
 	///Check if the event will end badly
 	var/is_zk_scenario = TRUE
+	///Ticks that have to pass before second wave
+	var/second_wave = 310
+	///Store the huge portals
+	var/obj/structure/crystal_portal/huge/center_portal
```

Comment:
Where is this reference being cleaned?

Reply:
right i forgot, i'll do the same as the crystal with the crystal and signals

Reply:
ah wait maybe i don't need to do a signal since it's only used in one proc i can clean it at the end of the proc

Reply:
nevermind, i don't even need that var in the first place